docs/docs/reference/release-notes/v2-maintenance-plan.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Gatsby Framework Version Support
+---
+
+## Overview
+
+As we continue evolving the Gatsby framework, and as we move from one major version to the next, we will communicate the duration and types of support provided to the prior major version. This allows users to effectively plan their Gatsby framework upgrades.
+
+## What is happening, when?
+
+Gatsby Version 2.32 will enter Maintenance Long Term Support on **June 8, 2021**. At that same time, Gatsby Version 3 will be considered the Active Long Term Support version of Gatsby. The Maintenance Long Term Support will continue until the release of Gatsby v4 (date TBD).
```

Comment:
Reorganizing the paragraph to try and group things together. Feel free to make adjustments.

It might also be worth it to briefly explain the difference between "maintenance long-term support" and "active long-term support". What's the difference in terms of what users can expect from us for each case?

```suggestion
Starting on **June 8, 2021**:

* Gatsby Version 3 will be considered the Active Long-Term Support version of Gatsby.
    * This means that ???.
* Gatsby Version 2.32 will enter Maintenance Long-Term Support.
    * This means that ???.
    * The Maintenance Long-Term Support will continue until the release of Gatsby v4 (date TBD).
```

Reply:
Yeah I love that, much more readable, and better structure. Will fill in the blanks. Thanks @meganesu